srtp/srtp.c

```diff
@@ -2568,9 +2568,9 @@ srtp_err_status_t srtp_unprotect_mki(srtp_ctx_t *ctx,
 
     /* Determine if MKI is being used and what session keys should be used */
     if (use_mki) {
-        session_keys = srtp_get_session_keys(
-            stream, (uint8_t *)hdr, (const unsigned int *)pkt_octet_len,
-            &mki_size);
+        session_keys =
+            srtp_get_session_keys(stream, (const uint8_t *)hdr,
+                                  (unsigned int)*pkt_octet_len, &mki_size);
```

Comment:
While this cast might be necessary, why do we have a mix of signed and unsigned integer use? That might be a good thing to harmonize. I personally prefer unsigned integers for length values.

Reply:
This is something we have been looking at and yes we need to harmonize on this and other types, it is coming :)
For length I would even go as far as using size_t every where including in the public api in v3!